scripts/west_commands/runners/jlink.py

```diff
@@ -39,6 +39,10 @@ def __init__(self, cfg, device,
         self.gdb_port = gdb_port
         self.tui_arg = ['-tui'] if tui else []
 
+        self.tool_opt = []
+        for opts in [opt.split() for opt in tool_opt]:
```

Comment:
I think this ought to be shlex.split.

```
In [1]: import shlex                                                                                     

In [2]: shlex.split("'--hello=\"my world\"'")                                                            
Out[2]: ['--hello="my world"']

In [3]: "'--hello=\"my world\"'".split()                                                                 
Out[3]: ['\'--hello="my', 'world"\'']
```

Reply:
Nice! I didn't know about shlex. Thanks!